### ISSUE TITLE
[BUG] Plugin removes default errors and warnings from Dart analyzer

### ISSUE THREAD

**Issue by mrRedSun** [dart-code-checker/dart-code-metrics#215]
* **Dart code metrics version:** 2.4.0
* **Dart sdk version:** 2.10.5 (stable)
* **OS: MacOs**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
include: package:lint/analysis_options.yaml

analyzer:
  strong-mode:
    implicit-casts: true
  exclude:
    - build/**
    - bin/**
    - ios/**
    - android/**
  plugins:
    - dart_code_metrics
  errors:
    missing_required_param: error
    missing_return: error
    must_be_immutable: error
    switch_expression_not_assignable: warning #todo: this should error, change level and fix build issues
    non_bool_condition: warning #todo: this should error, change level and fix build issues
    always_declare_return_types: warning #todo: this should error, change level and fix build issues
    invalid_override: warning #todo: this should error, change level and fix build issues
    getter_not_assignable_setter_types: warning #todo: this should error, change level and fix build issues
    sort_unnamed_constructors_first: ignore
    argument_type_not_assignable: ignore
    avoid_classes_with_only_static_members: ignore

# documentation https://pub.dev/documentation/dart_code_metrics/latest/
dart_code_metrics:
  anti-patterns:
    - long-method
    - long-parameter-list
  metrics:
    cyclomatic-complexity: 20
    lines-of-executable-code: 1000 #todo:change it to lower number
    number-of-arguments: 4
    maximum-nesting: 5
    break_on: 3
  metrics-exclude:
    - test/**
    - test_driver/**
    - ios/**
    - android/**
  rules:
    newline-before-return:
    no-empty-block:
    prefer-trailing-comma:
      break_on: 3

linter:
  rules:
    - annotate_overrides
    - await_only_futures
    - camel_case_types
    - cancel_subscriptions
    - close_sinks
    - constant_identifier_names
    - control_flow_in_finally
    - empty_statements

```

</details>

**What did you expect to happen?**
New issues from the plugin would be added alongside issues from the Dart analyzer
**What actually happened?**
Only issues from the plugin are shown

**Comment by incendial** [comment]
@mrRedSun hi! What IDE are you using? When issues from the Dart analyzer disappear, after reload?

**Comment by mrRedSun** [comment]
I'm using VSCode, I see the same picture on every restart, it first shows my issues from the Dart analyzer, and then they all disappear and issues from metrics start appearing

**Comment by incendial** [comment]
Hm, strange. They recently added lsp mode for the analyzer, can you please check `Preview LSP` dart setting in VSCode?

**Comment by mrRedSun** [comment]
I tried with Preview LSP on and off, didn't have an effect

**Comment by incendial** [comment]
I have tried to reproduce the problem, but with provided analysis_options config it works for me as expected. 

It would be very helpful if you could create an example where the issue could be reproduced.

**Comment by mrRedSun** [comment]
CLI worked as expected.
I just created the scratch project and it worked fine there, also the previous project works fine as well, it seems that the issues are VSCode and not plugin related.
Sorry for the confusion and waste of your time, closing the issue.

**Comment by incendial** [comment]
No problem, please let me know if the issue will appear again.
